README.md

```diff
@@ -242,3 +306,29 @@ class UserResource extends at.Resource {
 
 ***
 
+### Decorator 
+
+Angular provide a decorator feature to extends a service/factory. Use @decorator to use them and adding some others methods :
+
+```typescript
+@decorator('ngModuleName', 'SomeService')
+class SomeServiceDecorator {
+
+    constructor(
+        @at.inject('SomeService') private parent:SomeService;
+    ) {
+        // do stuff
+    }
+    
+    public someMethod2(anArg: number): boolean {
+        // do some stuff
+        
+       console.log(this.parent.someMethod(anArg));
```

Comment:
Maybe let's use some convention here - `parent` is a common word and it may cause collision. Maybe `$host` or `$delegate` would be better?